rust/relay/ebpf-turn-router/build.rs

```diff
@@ -12,6 +10,17 @@ use which::which;
 ///
 /// [bindeps]: https://doc.rust-lang.org/nightly/cargo/reference/unstable.html?highlight=feature#artifact-dependencies
 fn main() {
-    let bpf_linker = which("bpf-linker").expect("bpf-linker not found in $PATH");
-    println!("cargo:rerun-if-changed={}", bpf_linker.to_str().unwrap());
+    // eBPF is Linux-only, skip build script on other platforms
+    #[cfg(not(target_os = "linux"))]
```

Comment:
I reckon lets make the entire `main` fn conditional, then you don't need the `return` statement and we get a compile error if we ever mess up the `cfg` (because two mains are not allowed).